packages/component/src/Attachment/Text/private/shortenURL.ts

```diff
@@ -0,0 +1,17 @@
+export default function shortenURL(urlString: string): string {
```

Comment:
Would `extractHostnameWithSubdomain` be more accurate?

Reply:
Updating it.